Make reapply_sysctl ignore configs from /usr 

The reapply_sysctl functionality was originally added to address rhbz#1302953,
which was about Tuned overriding sysctl config files from /etc, which was
deemed undesirable. The bug said nothing about sysctl config files from
/usr/lib/sysctl.d, /lib/sysctl.d or /usr/local/lib/sysctl.d.

Having sysctl configs from /usr override Tuned profiles causes problems
(rhbz#1759597). Also, it seems reasonable for Tuned profiles to override
sysctl configs from /usr - a Tuned profile is often explicitly chosen by
the user, whereas sysctl configs from /usr are provided by system packages
whether the user wants them or not.

Also, in RHEL-6, tuned used to ignore sysctl config files from /usr, as far
as I can tell.

Resolves rhbz#1759597

See: https://github.com/redhat-performance/tuned/pull/226

diff --git a/tuned/plugins/plugin_sysctl.py b/tuned/plugins/plugin_sysctl.py
index b298bfa8..ee7596d5 100644
--- a/plugins/plugin_sysctl.py
+++ b/plugins/plugin_sysctl.py
@@ -12,10 +12,7 @@
 
 DEPRECATED_SYSCTL_OPTIONS = [ "base_reachable_time", "retrans_time" ]
 SYSCTL_CONFIG_DIRS = [ "/run/sysctl.d",
-		"/etc/sysctl.d",
-		"/usr/local/lib/sysctl.d",
-		"/usr/lib/sysctl.d",
-		"/lib/sysctl.d" ]
+		"/etc/sysctl.d" ]
 
 class SysctlPlugin(base.Plugin):
 	"""
